terraform/main.tf

```diff
@@ -19,10 +30,67 @@ resource "aws_vpc" "devops" {
   }
 }
 
+# Define Bastion inside the public subnet
```

Comment:
why don't you use the module structure?

Reply:
it's not special tool then no need to move